dev-requirements.txt

```diff
@@ -0,0 +1,22 @@
+# Formatters
+black
+isort
+
+# Linters
+dep-check
+mypy
+openapi-spec-validator
+pre-commit
+pyenchant
+pylint
+md-toc
+yamllint
```

Comment:
No

Reply:
Is it enough to have it in precommit ?

Reply:
I think you can remove the yamllint check, there are not yaml files in the repo

Reply:
.pre-commit.yml and action.yml